### PR TITLE
CallKit integration guide

### PR DESCRIPTION
**Pull request by x-0o0**
This guide has purpose on understanding CallKit framework with simple project.

New Added: `How to develop VoIP app with CallKit/README.md`

Only links and assets are still in progress

### CODE REVIEW

File: How to develop VoIP app with CallKit/README.md [hunk 1]
```diff
@@ -0,0 +1,472 @@
+# Table of Content
+- [Make Local Calls with CallKit](#make-local-calls-with-callkit)
+    * [Section1: The basic configuration of CallKit](#section1-the-basic-configuration-of-callkit)
+    * [Section2: Design CallKit UI](#section2-design-callkit-ui)
+    * [Section3: Request CallKit action](#section3-request-callkit-action)
+    * [Section4: Manage Call](#section4-manage-call)
+    * [Section5: Handling CallKit Events](#section5-handling-callkit-event)
+    * [Section6: Handle CXCallAction event](#section6-handle-cxcallacton-event)
```
Comment (mininny) [inline How to develop VoIP app with CallKit/README.md]: These two links are not working

Comment (x-0o0) [reply How to develop VoIP app with CallKit/README.md]: @mininny Thanks!!! I fixed it ([Fix typo](https://github.com/sendbird/guidelines-ios/pull/1/commits/2d8035eff678ed769519926dcbcef791f093de09))

File: How to develop VoIP app with CallKit/README.md [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+
+## Section6: Handle CXCallAction event 
+
+### (Interaction with CallKit UI)
+
+When the provider performs `CXCallActions`, corresponding `CXProviderDelegate` methods can be called. In order to properly respond to the users’ actions, you have to implement appropriate Sendbird Calls actions in the method.
+
+> **Important**
+> Don’t forget to execute `action.fulfill()` before the method is ended.
+
```
Comment (jaesung-0o0) [inline How to develop VoIP app with CallKit/README.md:347]: > **Important**
> To access `UUID` of the call, you have to use `action.callUUID` property, not `action.uuid`

Comment (x-0o0) [inline How to develop VoIP app with CallKit/README.md:347]:
```suggestion

> **Important**
> To access `UUID` of the call, you have to use `action.callUUID` property, not `action.uuid`

```

**Pull request merged**